04week/loop.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+
+
+/*for loop
+Use a for loop to console.log each item in the array carsInReverse.*/
+ const carsInReverse = () => {
+   const carsInReverse = ['Ford', 'Chevy', 'Toyota', 'Honda'];
+ for (let i = 0; i < carsInReverse.length; i++) {
+    console.log(carsInReverse[i]) + "<br>";
+ }
+}
+console.log(carsInReverse());
+
+/*for...in loop
+Create an object (an array with keys and values) called persons with the following data:
+firstName: "Jane"
+lastName: "Doe"
+birthDate: "Jan 5, 1925"
+gender: "female"
+Use a for...in loop to console.log each key.
+Then use a for...in loop and if state to console.log the value associated with the key birthDate.
+*/
+const persons = {firstName: "Jane", lastName: "Doe", birthDate: "Jan 5, 1925", gender: "female"};
+
+console.log(Object.keys(persons));
+console.log(Object.values(persons));
+for (const prop in persons){
+ console.log(`obj.${prop} = ${persons[prop]}`);
```

Comment:
putting obj.${prop} like that will print out obj.firstName = "Jane". so fix that.